tutorials/video_recorder.md

```diff
@@ -0,0 +1,107 @@
+\page videorecorder Video Recorder
+
+## Using the video recorder plugin
+
+Ignition Gazebo offers an video recorder tool for recording videos from the 3D
+Scene. The tool is available as a GUI plugin. To open this plugin, first launch
+Ignition Gazebo and select the ellipsis menu on top right (3 dots menu), and
+scroll down to find the `Video Recorder` option. Click on the plugin to open
+the Video Recorder tool. Alternatively, launch the demo world in Ignition Gazebo
+that already has this plugin included in the GUI.
+
+```
+ign gazebo -v 4 video_record_dbl_pendulum.sdf
+```
+
+In this plugin, you should see a single button with a video recorder icon.
+Clicking on the button gives you the video format options that are available.
+
+![Video Recorder formats](https://github.com/ignitionrobotics/ign-gazebo/raw/7dccc0a08f9e926b9460c7662d49dbb0ce24ea61/tutorials/files/video_recorder/video_recorder.png)
```

Comment:
I recently learned that we can refer to images using `@image html` like this:

https://github.com/ignitionrobotics/ign-gazebo/blame/ign-gazebo4/tutorials/gui_config.md#L38

Reply:
updated. 46a7d1f

Is there a way to preview it? it does not seem to work locally and on github though.

Reply:
Yup you can `make doc` and see the doxygen-generated tutorials on a browser

Reply:
ah I just had to clear the build cache and rebuild and now it's showing up locally.